can_testbench.py

```diff
@@ -1,23 +1,27 @@
+# nuitka-project: --enable-plugin=pyside6
+# nuitka-project: --disable-console
+# nuitka-project: --standalone
+# nuitka-project: --include-module=can.interfaces.slcan
+# nuitka-project: --include-module=can.interfaces.udp_multicast
```

Comment:
Have to manually add any new can.interfaces we implement here because python-can uses dynamic imports for the interfaces which aren't automatically detected by Nuitka